### ISSUE TITLE
Warning message and freezeup on Windows 10 / node 7.1.0

### ISSUE THREAD

**Issue by christav** [Microsoft/mobile-center-cli#62]
When attempting to login on a Windows 10 box running node 7.1, you get:
```
> mobile-center login
(node:13360) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: This socket is closed
(node:13360) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
and then the CLI freezes up for quite a while waiting for some sort of timeout.

Works fine on Node 6.9.

**Comment by christav** [comment]
This appears to be a regression in node, see https://github.com/nodejs/node/issues/9542.
Filing this issue here for tracking, won't be attempting further fix / workaround.

This broke a lot more than just us - yeoman, mocha, inquirer in general have all reported same error.

**Comment by christav** [comment]
Fixed in node 7.2.0
